odl/contrib/datasets/images/cambridge.py

```diff
@@ -0,0 +1,151 @@
+
+    Output:
+        image type: gray scales
+        size: [1024, 1024] (if not specified by `size`)
+        scale: [0, 1]
+        type: float64
+    """
+    # TODO: Store data in some ODL controlled url
+    name = 'PET_phantom.mat'
+    url = URL_CAM + name
+    dct = get_data(name, subset=DATA_SUBSET, url=url)
+    im = np.rot90(dct['im'], k=3)
+
+    return convert(im, size, to_odl)
+
+
+def resolution_phantom(size=None, to_odl=True):
+    """Resolution phantom for tomographic simulations.
+
+    Output:
+        image type: gray scales
+        size: [1024, 1024] (if not specified by `size`)
+        scale: [0, 1]
+        type: float64
+    """
+    # TODO: Store data in some ODL controlled url
+    # TODO: This can be also done with ODL's ellipse_phantom
+    name = 'phantom_resolution.mat'
+    url = URL_CAM + name
+    dct = get_data(name, subset=DATA_SUBSET, url=url)
+    im = np.rot90(dct['im'], k=3)
+
+    return convert(im, size, to_odl)
+
+
+def building(size=None, gray=False, to_odl=True):
+    """Photo of the Centre for Mathematical Sciences in Cambridge.
+
+    Output:
+        image type: color (or gray scales if `gray=True`)
+        size: [442, 331] (if not specified by `size`)
+        scale: [0, 1]
+        type: float64
+    """
+    # TODO: Store data in some ODL controlled url
+    name = 'cms.mat'
+    url = URL_CAM + name
+    dct = get_data(name, subset=DATA_SUBSET, url=url)
+    im = np.rot90(dct['im'], k=3)
+
+    return convert(im, size, to_odl, gray=gray)
+
+
+def rings(size=None, gray=False, to_odl=True):
+    """Photo of married couple holding hands.
+
+    Output:
+        image type: color (or gray scales if `gray=True`)
+        size: [3264, 2448] (if not specified by `size`)
+        scale: [0, 1]
+        type: float64
+    """
+    # TODO: Store data in some ODL controlled url
+    name = 'rings.mat'
+    url = URL_CAM + name
+    dct = get_data(name, subset=DATA_SUBSET, url=url)
+    im = np.rot90(dct['im'], k=2)
+
+    return convert(im, size, to_odl, gray=gray)
+
+
+def blurring_kernel(size=None, to_odl=True):
+    """Blurring kernel for convolution simulations.
+
+    The kernel is scaled to sum to one.
+
+    Output:
+        image type: gray scales
+        size: [100, 100] (if not specified by `size`)
+        scale: [0, 1]
+        type: float64
+
+    """
+    # TODO: Store data in some ODL controlled url
+    name = 'motionblur.mat'
+    url = URL_CAM + name
+    dct = get_data(name, subset=DATA_SUBSET, url=url)
+
+    return convert(255 - dct['im'], size, to_odl, normalize='sum')
+
+
+def convert(image, size, to_odl, gray=None, dtype='float64', normalize='max'):
+
+    image.astype(dtype)
+
+    if gray is not None and gray:
+        image = np.mean(image, 2)
+
+    if size is not None:
+        image = imresize(image, size)
+
+    if normalize == 'max':
+        image = image / image.max()
+
+    elif normalize == 'sum':
+        image = image / image.sum()
+
+    if to_odl:
```

Comment:
I would personally vote against a `to_odl` function since it can be a bit confusing for users. Explicitly writing

```
space.element(cambridge.building())
```

is (imo) more clear.

Reply:
You mean that all images are converted to ODL without the flag or that no images are converted to ODL?

Reply:
I would suggest that no images are converted, e.g. that this simply returns numpy arrays.

Reply:
OK, done.